src/components/CardBase/CardBase.tsx

```diff
@@ -1,4 +1,11 @@
-import React, {Children, Fragment, HTMLAttributeAnchorTarget, ReactElement} from 'react';
+import React, {
+    Children,
+    Fragment,
+    HTMLAttributeAnchorTarget,
+    PropsWithChildren,
+    ReactElement,
+    isValidElement,
```

Comment:
what is the purpose of `isValidElement`? didn't find any usage examples in pr

Reply:
Please have a look at line 96. It used to handle children elements correctly.